framework/test_app/runners/k8s/k8s_xds_client_runner.py

```diff
@@ -97,13 +96,6 @@ def __init__(  # pylint: disable=too-many-locals
         self.debug_use_port_forwarding = debug_use_port_forwarding
 
         # Client deployment arguments.
-        if not deployment_args:
-            deployment_args = ClientDeploymentArgs(
-                # TODO(stanleycheung): remove once https://github.com/grpc/psm-interop/pull/33
-                #    is merged and removed self.csm_* removed as class args.
-                csm_workload_name=csm_workload_name,
-                csm_canonical_service_name=csm_canonical_service_name,
-            )
```

Comment:
Looks like that's the problem. It should be 
```py
        if not deployment_args:
            deployment_args = ClientDeploymentArgs()
```

Similar to https://github.com/grpc/psm-interop/blob/70348eae3e9106c594eaa2743909f8739ce6191d/framework/test_app/runners/k8s/k8s_xds_server_runner.py#L123-L124.

Sorry for the confusion, `TODO(stanleycheung): remove once ...` was only in regards to `csm_workload_name`/`csm_canonical_service_name` params, not the whole statement.

Reply:
Fixed. Running the test suites again